### ISSUE TITLE
Box matching loss used in training KOSMOS-2

### ISSUE THREAD

**Issue by arijitray1993** [microsoft/unilm#1190]
Hi, I had a question about the KOSMOS-2 model. How is the loss calculated when there are multiple instances of the same object (or text span)? Do you use Hungarian matching loss in these cases? Or is there a fixed ordering in which the boxes are predicted?

**Comment by pengzhiliang** [comment]
Hi, @arijitray1993 
KOSMOS-2 does not use the Hungarian matching algorithm, but calculates cross-entropy losses based on the order of box in the dataset. No other tricks are employed here.
